components/button/index.css

```diff
@@ -10,79 +10,117 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/scales/spectrum-large.css";
-@import "../vars/css/scales/spectrum-medium.css";
 @import "../commons/basebutton.css";
```

Comment:
I think we need to remove this line and only import the `basebutton-coretokens.css` file.